addons/controls/src/ControlsPanel.tsx

```diff
@@ -36,6 +37,7 @@ export const ControlsPanel: FC = () => {
       {showWarning && <NoControlsWarning />}
       <ArgsTable
         {...{
+          key: path, // resets state when switching stories
```

Comment:
Would `storyId` make more sense here?

Reply:
Yeah I did that at first but it would then have to be:

```
key: refId ? `${refId}_${storyId}` : storyId
```

`path` achieves the same thing but without the added complexity of dealing with `refId` (it's in the path already).